core/os/midi_driver.cpp

```diff
@@ -43,76 +43,82 @@ void MIDIDriver::set_singleton() {
 }
 
 void MIDIDriver::receive_input_packet(int device_index, uint64_t timestamp, uint8_t *data, uint32_t length) {
```

Comment:
Since we're rewriting this let's fix the arguments
```suggestion
void MIDIDriver::receive_input_packet(int p_device_index, uint64_t p_timestamp, uint8_t *p_data, uint32_t p_length) {
```